hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Filter/ReviewOrgUnitChangesFilter.tsx

```diff
@@ -57,6 +83,43 @@ export const ReviewOrgUnitChangesFilter: FunctionComponent<Props> = ({
             })) || [],
         [forms],
     );
+    const initialDataSource = useMemo(
+        () =>
+            dataSources?.find(
+                source =>
+                    source.value === defaultSourceVersion.source.id.toString(),
+            )?.value || '',
+        [dataSources, defaultSourceVersion.source.id],
+    );
+
+    const [showAdvancedSettings, setShowAdvancedSettings] = useState(false);
+    const { formatMessage } = useSafeIntl();
+
+    const {
+        data: groupOptions,
+        isLoading: isLoadingGroups,
+        refetch: refetchGroups,
+    } = useGetGroupDropdown(
+        selectedVersionId ? { defaultVersion: selectedVersionId } : {},
+    );
+
+    useEffect(() => {
+        if (selectedVersionId) {
+            refetchGroups();
+        }
+    }, [selectedVersionId, refetchGroups]);
```

Comment:
Is there a reason you don't pass `selectedVersionId` to `useGetGroupsDropdown` and put it in the query key instead of this `useEffect` ?